code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -143,6 +146,13 @@
 	recovery_time = world.time + buffer_time
 	ranged_cooldown = world.time + buffer_time
 
+/mob/living/simple_animal/hostile/megafauna/proc/do_chasm()
+	for(var/turf/simulated/floor/chasm/C in circlerangeturfs(src, 1))
+		if(!C.density)
+			continue
+		C.density = FALSE //I hate it.
+		addtimer(VARSET_CALLBACK(C, density, TRUE), 2.25 SECONDS) // Needed to make them path. I hate it.
```

Comment:
Why this number in particular, and why did you increase it here?

Reply:
Life ticks every 2 seconds. If the callback is 2 seconds, the density resets before the logic decides to move

Reply:
Can you add a comment indicating exactly that, so it seems like less of a magic number?

Reply:
Sure!